recipe_scrapers/amazingoriental.py

```diff
@@ -0,0 +1,30 @@
+from ._abstract import AbstractScraper
+
+
+class AmazingOriental(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "amazingoriental.com"
+
+    def title(self):
+        return self.soup.find('h1').get_text()
+
+    def category(self):
+        return self.soup.select("div.recipe-info .meal-type")[0].get_text()
+
+    def total_time(self):
+        return self.soup.select("dl.prepare-time dd")[0].get_text()
+
+    def yields(self):
+        return self.soup.select(".person-amount")[0].get_text()
+
+    def ingredients(self):
+        ingredients = self.soup.select(".sidebar .ingredients-wrap ul li")
+        return [ingredient.get_text() for ingredient in ingredients]
```

Comment:
```suggestion
        return [' '.join(ingredient.get_text().split()) for ingredient in ingredients]
```
The output of the ingredients can be improved with this change, this will remove the double space between unit and ingredient as well as strip the empty space at the end of the strings.